### DISCUSSION TITLE
yarn lint after init constantly says `ESLint must be installed: yarn add --dev eslint`

### DISCUSSION THREAD

**Discussion by The-Code-Monkey** [vercel/next.js#33341] [Help]
### Run `next info` (available from version 12.0.8 and up)
```js
{
  name: 'portfolio',
  description: 'Portfolio library for Node.js',
  'dist-tags': {
    latest: '0.1.0'
  },
  versions: [
    '0.1.0'
  ],
  maintainers: [
    {
      name: 'pilwon',
      email: 'pilwon@gmail.com'
    }
  ],
  time: {
    modified: '2017-01-19T00:38:53.450Z',
    created: '2013-11-27T06:10:36.402Z',
    '0.1.0': '2013-11-27T06:10:37.553Z'
  },
  author: {
    name: 'Pilwon Huh',
    email: 'pilwon@gmail.com'
  },
  repository: {
    type: 'git',
    url: 'git://github.com/pilwon/node-portfolio.git'
  },
  homepage: 'https://github.com/pilwon/node-portfolio',
  keywords: [
    'finance',
    'portfolio',
    'stock',
    'ofx'
  ],
  bugs: {
    url: 'https://github.com/pilwon/node-portfolio/issues'
  },
  license: 'MIT',
  readmeFilename: 'README.md',
  version: '0.1.0',
  dependencies: {
    async: '~0.2.9',
    ent: '~0.1.0',
    lodash: '~2.3.0',
    moment: '~2.4.0',
    string: '~1.7.0',
    xml2js: '~0.4.0'
  },
  devDependencies: {
    colors: '~0.6.2',
    grunt: '~0.4.2',
    'grunt-contrib-jshint': '~0.7.2',
    'grunt-contrib-watch': '~0.5.3',
    matchdep: '~0.3.0'
  },
  scripts: {
    prepublish: 'npm prune'
  },
  dist: {
    shasum: 'e6b1208b918f9823f86fca0ebac4457da5981dd9',
    tarball: 'https://registry.npmjs.org/portfolio/-/portfolio-0.1.0.tgz'
  },
  directories: {}
}
```

### What version of Next.js are you using?

latest

### What version of Node.js are you using?

12.22.8

### What browser are you using?

firefox

### What operating system are you using?

linux

### How are you deploying your application?

vercel

### Describe the Bug

you run yarn lint to set up eslint then i selected strict.

i then run yarn lint again and all i get is 

`ESLint must be installed: yarn add --dev eslint`

which is incorrect as next just installed it all. I have cleared node modules and started again but nothing.

### Expected Behavior

should run the linter

### To Reproduce

I have nothing custom running apart from Nextra which is built by a couple of guys who work for vercel.

**Comment by balazsorban44** [comment]
I can see from your `package.json` that you have not installed `eslint`. Please follow the given error and run `yarn add --dev eslint`

**Reply by The-Code-Monkey** [reply]
I think the problem is that next js isnt picking it up as my package json contains eslint and so does my node modules

**Comment by The-Code-Monkey** [comment]
![image](https://user-images.githubusercontent.com/31218527/149622115-1827df29-87d5-4e29-8874-879da3e38d18.png)

as you can see its installed

**Reply by The-Code-Monkey** [reply]
![image](https://user-images.githubusercontent.com/31218527/149622131-a81e18b5-cc18-4b5d-9c5e-fcd44cdcb2fa.png)
yet nextjs give me this, and yes its the same project in the terminal as in the window

**Reply by balazsorban44** [reply]
`@eslint` isn't the same as `eslint`

**Comment by enero-o** [comment]
downgrade to version 7  
` "eslint": "^7.32.0",`
